changelog.md

```
@@ -1,4 +1,12 @@
 # acts_as_trackable Changelog
+## Version: 0.4.11
+  ### Patch
```

markdownlint-cli2 (0.23.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)